worlds/raft/__init__.py

```diff
@@ -114,6 +114,14 @@ def set_rules(self):
     def create_regions(self):
         create_regions(self.multiworld, self.player)
     
+    def get_pre_fill_items(self):
```

Comment:
Raft's `pre_fill` has been removed as of https://github.com/ArchipelagoMW/Archipelago/commit/a948697f3a2387d13862a194690c39af8da2dbd8